src/CMakeLists.txt

```diff
@@ -828,8 +838,28 @@ if (LIBRT)
         )
 endif ()
 
+if (BUILD_WITH_CRASHPAD)
+    target_compile_definitions(${LIBRARY_PROJECT} PUBLIC CHATTERINO_WITH_CRASHPAD)
+    target_link_libraries(${LIBRARY_PROJECT} PUBLIC crashpad::client)
+    set_target_directory_hierarchy(crashpad_handler crashpad)
+endif()
+
 # Configure compiler warnings
 if (MSVC)
+    # Change flags for RelWithDebInfo
+
+    # Default: "/debug /INCREMENTAL"
+    # Changes:
+    #   - Disable incremental linking to reduce padding
+    #   - Enable all optimizations - by default when /DEBUG is specified,
+    #     these optimizations will be disabled. We need /DEBUG to generate a PDB.
+    set(CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO "/DEBUG /INCREMENTAL:NO /OPT:REF,ICF,LBR")
```

Comment:
The changes are based on [this issue](https://gitlab.kitware.com/cmake/cmake/-/issues/20812).

Reply:
Can we link to the issue in the comment too?